Userland/Libraries/LibWeb/CSS/Parser/Token.h

```diff
@@ -111,10 +111,15 @@ class Token {
         VERIFY(m_type == Type::Number);
         return m_value.string_view();
     }
+    double number_value() const
+    {
+        VERIFY(m_type == Type::Number);
+        return m_number_value;
+    }
     i64 to_integer() const
     {
         VERIFY(m_type == Type::Number && m_number_type == NumberType::Integer);
-        return number_string_value().to_int<i64>().value();
+        return static_cast<i64>(number_value());
```

Comment:
what if it no fits in `i64`?

Reply:
Turns out the spec says to round things, which I hadn't seen before:
> The precision and supported range of numeric values in CSS is explicitly undefined, and can vary based on the property or other context a value is used in. However, within the CSS specifications, infinite precision and range is assumed. When a value cannot be explicitly supported due to range/precision limitations, it must be converted to the closest value supported by the implementation, but how the implementation defines "closest" is explicitly undefined as well.

So, I'll do that.